locales/en-US/app.ftl

```diff
@@ -102,6 +103,84 @@ ListOfPublishedProfiles--uploaded-profile-information-list =
        *[other] Manage these recordings
     }
 
+## MenuButtons Folder
+
+MenuButtons--index--profile-info-uploaded-label = Uploaded:
+MenuButtons--index--profile-info-uploaded-actions = Delete
+MenuButtons--index--meta-info-subtitle = Profile Information
+
+# Used in the meta info menu button, but the label is changed depending on the variable
+# Variables:
+#   $labelVariable (String) - Label for meta info button based on republish and error status
+MenuButtons--index--meta-info-button =
+    .label = { $labelVariable }
```

Comment:
Sorry, I don't understand the comment. Why do we need to expose a string for this? We should manage this in the code, without asking all locales to "translate" this message.